lib/ansible/galaxy/role.py

```diff
@@ -115,16 +115,18 @@ def install_info(self):
         """
         if self._install_info is None:
 
-            info_path = os.path.join(self.path, self.META_INSTALL)
-            if os.path.isfile(info_path):
-                try:
-                    f = open(info_path, 'r')
-                    self._install_info = yaml.safe_load(f)
-                except Exception:
-                    display.vvvvv("Unable to load Galaxy install info for %s" % self.name)
-                    return False
-                finally:
-                    f.close()
+            for path in self.paths:
+                info_path = os.path.join(path, self.META_INSTALL)
+                if os.path.isfile(info_path):
+                    try:
+                        f = open(info_path, 'r')
+                        self._install_info = yaml.safe_load(f)
+                        self.path = path
+                    except Exception:
+                        display.vvvvv("Unable to load Galaxy install info for %s" % self.name)
+                        return False
+                    finally:
+                        f.close()
```

Comment:
I would not do this by default, this should be a new extra options --show-all-isntalled or similar

Reply:
This is not to show, but to actually install. The fact is that the roles path defined at ansible.cfg is basically ignored when installing using `ansible-galaxy`